vexim/config/tests/functionsValidatePasswordTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+	
+class FunctionValidatePasswordTest extends PHPUnit_Framework_TestCase {
+		
+	public function setUp() {
+		require_once(__DIR__ . '/../functions.php');
+	}
+
+        public function testSendEmptyPasswords()
+        {
+            $this->assertFalse(validate_password("", ""));
+        }
+
+        public function testSendSamePasswords()
+        {
+            $this->assertTrue(validate_password("password", "password"));
+        }
+
+        public function testSendDifferentPasswords()
+        {
+            $this->assertFalse(validate_password("password", "pass"));
+        }
+
+        public function testSendNullValues()
+        {
+            $this->assertFalse(validate_password(null, null));
+        }
+
+        public function testSendBoolValues()
+        {
+            $this->assertFalse(validate_password(true, true));
+            $this->assertFalse(validate_password(false, false));
+        }
+
+        public function testSendIntegerValues()
+        {
+            $this->assertFalse(validate_password(10, 10));
+        }
+
+        public function testSendZeroValues()
+        {
+            $this->assertFalse(validate_password(0, 0));
+        }
+}
+
+### EOF
```

Comment:
Just an End Of File comment I add to my PHP files in order to omit **?>**
> "If a file is pure PHP code, it is preferable to omit the PHP closing tag at the end of the file. This prevents accidental whitespace or new lines being added after the PHP closing tag, which may cause unwanted effects because PHP will start output buffering when there is no intention from the programmer to send any output at that point in the script."
> http://php.net/manual/en/language.basic-syntax.phptags.php
But I add this comment to mark the end of the file so I know where to stop scrolling down when I read the code. 

It's just a personal preference, it's definitely not necessary and can be removed if required.